Tests/Transfer/HyperwalletTransferTests.swift

```diff
@@ -0,0 +1,120 @@
+        if Hippolyte.shared.isStarted {
+            Hippolyte.shared.stop()
+        }
+    }
+
+    func testCreateTransfer_success() {
+        // Given
+        let expectation = self.expectation(description: "Create transfer completed")
+        let response = HyperwalletTestHelper.okHTTPResponse(for: "CreateTransferResponse")
+        let url = String(format: "%@transfers", HyperwalletTestHelper.restURL)
+        let request = HyperwalletTestHelper.buildPostRequest(baseUrl: url, response)
+        HyperwalletTestHelper.setUpMockServer(request: request)
+
+        var transferResponse: HyperwalletTransfer?
+        var errorResponse: HyperwalletErrorType?
+
+        //When
+        let transferRequest = HyperwalletTransfer.Builder(clientTransferId: "6712348070812",
+                                                          sourceToken: "usr-123456",
+                                                          destinationToken: "trm-invaqlid-token")
+            .destinationAmount("62.29")
+            .destinationCurrency("USD")
+            .build()
+
+        Hyperwallet.shared.createTransfer(transfer: transferRequest, completion: { (result, error) in
+            transferResponse = result
+            errorResponse = error
+            expectation.fulfill()
+        })
+        wait(for: [expectation], timeout: 1)
+
+        // Then
+        XCTAssertNotNil(transferResponse, "The `transferResponse` should not be nil")
+        XCTAssertNil(errorResponse, "The `errorResponse` should be nil")
+
+        verifyCreateTransferResponse(transferResponse)
+    }
+
+    func testCreateTransfer_invalidDestinationToken_returnBadRequest() {
+        // Given
+        let expectation = self.expectation(description: "Create transfer failed")
+        let response = HyperwalletTestHelper
+            .badRequestHTTPResponse(for: "CreateTransferResponseInvalidDestinationToken")
+        let url = String(format: "%@transfers", HyperwalletTestHelper.restURL)
+        let request = HyperwalletTestHelper.buildPostRequest(baseUrl: url, response)
+        HyperwalletTestHelper.setUpMockServer(request: request)
+
+        var transferResponse: HyperwalletTransfer?
+        var errorResponse: HyperwalletErrorType?
+
+        // When
+        let transferRequest = HyperwalletTransfer.Builder(clientTransferId: "6712348070812",
+                                                          sourceToken: "usr-123456",
+                                                          destinationToken: "trm-invaqlid-token")
+            .destinationAmount("62.29")
+            .destinationCurrency("USD")
+            .build()
+
+        Hyperwallet.shared.createTransfer(transfer: transferRequest, completion: { (result, error) in
+            transferResponse = result
+            errorResponse = error
+            expectation.fulfill()
+        })
+        wait(for: [expectation], timeout: 1)
+
+        // Then
+        guard transferResponse == nil else {
```

Comment:
update assert statements to be more simplified like 
`XCTAssertNil(transferResponse, "The `transferResponse` should be nil")`
 instead of guard

Reply:
Fixed